netman/adapters/switches/cisco.py

```diff
@@ -89,6 +89,7 @@ def get_vlan(self, number):
             vlan,
             self.ssh.do("show running-config interface vlan {} | begin interface".format(number))
         )
+        apply_interfaces_data(vlan, self.get_interfaces())
```

Comment:
That can really slow the hell down this method that should be quick
Any other way not listing all interfaces?